docs/reference/changelog.rst

```diff
@@ -4,6 +4,15 @@
 Changelog
 *********
 
+4.4.0 (2024-Oct-XX)
```

Comment:
question: I guess the proper date will be set just before merging or was it forgotten?

Reply:
It's left on purpose because I don't know when it'll be (probably today). It'll be fixed in a follow-up PR